app/components-react/windows/settings/Hotkey.m.less

```diff
@@ -0,0 +1,27 @@
+@import '../../../styles/index.less';
+
+.hotkey {
+  display: flex;
+  flex-direction: row;
+  .margin-bottom(3);
+}
+
+.Hotkey-description {
+  width: 200px;
+  padding-right: 10px;
+}
+
+.Hotkey-input {
+  display: inline-block;
+  width: 150px;
+}
+
+.Hotkey-control {
```

Comment:
i assume these are capitalized because the old vue ones were but its not really a pattern we use these days